.github/workflows/ci.yml

```diff
@@ -45,12 +86,26 @@ jobs:
     - name: Run go generate
       run: go generate ./...
       working-directory: ${{ matrix.dir }}
+    # Download the custom linter binary created (or loaded from cache) in the builder-linter job above.
+    - name: Download custom linter
+      uses: actions/download-artifact@v4
+      with:
+        name: custom-gcl
+        path: ./tools/
```

Comment:
Can we leverage the cache for accessing the binary rather than uploading & downloading artifacts?